src/components/Sidebar/index.tsx

```diff
@@ -41,15 +41,31 @@ const search = debounce((router: NextRouter, searchTerm: string) => {
 
 const aggregateMemberLinks = (
   members: Member[],
-  field: string,
+  field: keyof Member,
   prefix: string
 ) => {
-  return Object.entries(groupBy(members, field))
+  const isArrayType = Array.isArray(members[0][field]);
+  let aggregateGroup: [fieldValue: string, group: Member[]][] = [];
+  if (isArrayType) {
+    let fieldAggregate: string[] = Array.from(
+      new Set(members.flatMap((member) => member[field]))
+    ) as string[];
+    aggregateGroup = fieldAggregate.map((fieldValue) => [
+      fieldValue,
+      members.filter((member) =>
+        (member[field] as string[]).includes(fieldValue)
+      ),
+    ]);
+  }
```

Comment:
This is some hairy logic, but ultimately it's aggregating the field values (like an organization or team name) with lists of members who have a matching field value.